src/node_sea.cc

```diff
@@ -248,7 +273,42 @@ ExitCode GenerateSingleExecutableBlob(const SeaConfig& config) {
     return ExitCode::kGenericUserError;
   }
 
-  SeaResource sea{config.flags, main_script};
+  std::vector<char> snapshot_blob;
```

Comment:
What do you think about moving the snapshot generation part into a separate function for more modularity?